backend/sportsmatch/src/main/java/com/sportsmatch/auth/AuthService.java

```diff
@@ -26,16 +23,18 @@ public class AuthService {
 
   public void register(AuthRequestDTO authRequestDTO) {
     User user = userMapper.registerToUser(authRequestDTO);
+    if (userRepository.existsByEmail(user.getEmail())) {
+      throw new ResponseStatusException(HttpStatus.FORBIDDEN);
```

Comment:
409 conflict would be better here, forbidden would mean the user doesn't have access to creating a new account